rider-junit5/src/main/java/com/github/database/rider/junit5/DBUnitExtension.java

```diff
@@ -282,6 +289,10 @@ private String getExecutorId(final ExtensionContext extensionContext, DataSet da
             annDataSet = findDataSetAnnotation(extensionContext);
         }
         String dataSourceBeanName = getConfiguredDataSourceBeanName(extensionContext);
+        return getExecutorId(dataSourceBeanName, annDataSet);
+    }
+
+    private String getExecutorId(String dataSourceBeanName, Optional<DataSet> annDataSet) {
```

Comment:
Please pass the dataset object and do the `Optional.of(dataSet)` inside the method, there's no guarantee that the optional method param is not null

Reply:
Thank for quick code review!

In common case the best practice is to not pass Optional as parameter, but in this case we have got Optional from AnnotationUtils.findAnnotation() first and pass it as parameter into getExecutorId(). Are you sure the best way here is to unwrap Optional before passing?

Reply:
I think we can keep as is